plugin/pkg/admission/webhook/gencerts.sh

```diff
@@ -44,11 +41,7 @@ distinguished_name = req_distinguished_name
 [req_distinguished_name]
 [ v3_req ]
 basicConstraints = CA:FALSE
-keyUsage = nonRepudiation, digitalSignature, keyEncipherment
```

Comment:
Could you explain why we need to remove this line?

Reply:
no, its a typo! i should not remove it.